### DISCUSSION TITLE
How to handle responsive styles in v9?

### DISCUSSION THREAD

**Discussion by AkiraVoid** [microsoft/fluentui#24222] [Q&A] [answered]
In v8, we got `useResponsiveMode`, is there any way we can use alternatively?

**Comment by micahgodbolt** [comment, marked as answer]
with v9 controls we are trying to not make as many assumptions as we did in v8, so none of our controls have built in responsive features. We recommend using whatever took works best for you. That could be raw media queries in your CSS, or an open source hook based approach like https://www.npmjs.com/package/@react-hook/media-query
